internal/repository/repository_public_test.go

```diff
@@ -74,6 +75,7 @@ func (suite *RepositoryPublicTestSuite) SetupTest() {
 	suite.cloneDir = "/cloneDir"
 	suite.dstDir = "/dstDir"
 	suite.gitURL = "https://example.com/user/repo.git"
+	suite.cacheDir = "https---example.com-user-repo.git"
```

Comment:
I know it's silly to explicitly test parts of the stdlib, but I'm feeling "better safe than sorry" these days